pkg/frontend/authenticate.go

```diff
@@ -3279,88 +3289,24 @@ func doCreateStage(ctx context.Context, ses *Session, cs *tree.CreateStage) (err
 func doCheckFilePath(ctx context.Context, ses *Session, ep *tree.ExportParam) (err error) {
 	//var err error
 	var filePath string
-	var sql string
-	var erArray []ExecResult
-	var stageName string
-	var stageStatus string
-	var url string
 	if ep == nil {
 		return err
 	}
 
-	bh := ses.GetBackgroundExec(ctx)
-	defer bh.Close()
-
-	err = bh.Exec(ctx, "begin;")
-	defer func() {
-		err = finishTxn(ctx, bh, err)
-	}()
-	if err != nil {
-		return err
-	}
-
 	// detect filepath contain stage or not
 	filePath = ep.FilePath
```

Comment:
The logic is changed in this PR.  There is no need to query the stage record and check it. Am I right?

Reply:
enable/disable in stage_status is not valid.  The path expansion is done in pkg/sql/plan/function/stage_util.go now.